src/locales/locale-en.json

```diff
@@ -298,5 +298,7 @@
   "already taken": "Already taken",
   "Enter a valid timezone": "Enter a valid timezone",
   "write a message": "Write a message...",
-  "write the first message": "Write the first message..."
+  "write the first message": "Write the first message...",
```

Comment:
I think we need to rationalize this locale approach, seems to be append-only lately...

Reply:
For sure, we also need to add a lot of untranslated stuff into locale-en...